docs/ChangeLog

```diff
@@ -1,3 +1,69 @@
+New in 2021.12
+
+Core:
++ Allocate frame work and, when possible, environment registers on the
+  callstack [fd571614][9266e794][7eca0ed3][d83edbea]
++ Negative numbers should not be prime [1bd61725]
++ Fix segfaults when a frame has more than 8192 locals [24392a32]
++ Some small optimizations for frame allocation [9903a543][fcddd121]
+  [4d145b46][93809811]
++ When NativeCall is libffi, copy `ffi_ret_type` and `ffi_arg_types`
+  [a2a82f99]
++ When NativeCall is libffi, don't leak ffi_arg_types during GC [a3c17d0f]
++ Avoid thread safety issues in intern lookups [08d41d5e]
++ Fix threading issues caused by misalignment in --valgrind builds
+  [ea5a86a1]
++ Remove counter productive NULL checks in decont ops [8aca849f]
++ Dispatcher replace arg syscall [68b8f429][ed0c8ec0][4cdce310][0a5b37a1]
++ Migrate special return to callstack and simplify return handling
+  [05b34685][485ca825][f58cf69f][139f54aa][feba1cd5][3f45b234][8e197406]
+  [776a1e3c][4adf3a2c][ce19abf2][0f43204e][2f423b20][b8e90bf8][cb7deb4b]
+  [843cf0f3][cd3b759b][f2685cd9][9f22ccc2]
++ Mark nqp::radix as :pure [9d5bcfc5]
++ New disp nativecall libffi [e3614b05][8b33b49a][1eb6bd4c][58cb30a8]
```

Comment:
It's not just libffi, so "New disp nativecall" is already correct.

Reply:
fixed and forced